app/assets/javascripts/discourse/models/user.js

```diff
@@ -840,6 +840,21 @@ const User = RestModel.extend({
       !secondFactorEnabled &&
       (enforce === "all" || (enforce === "staff" && staff))
     );
+  },
+
+  resolvedTimezone() {
+    if (this.timezone) {
```

Comment:
Minor but for private variables like this I'd suggest using `this._timezone` - it makes it obvious to someone building on top of this that they should be careful about using that variable rather than the `resolvedTimezone()` method.